src/CheckoutSdk.SampleApp/Views/Payment/Index.cshtml

```diff
@@ -0,0 +1,68 @@
+﻿@{
+    ViewData["Title"] = "Payment Page";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+@model CheckoutSdk.SampleApp.Models.PaymentModel
+
+<script src="https://cdn.checkout.com/js/frames.js"></script>
+<h2>Sample application</h2>
+
+<h4>New payment</h4>
+<hr />
+<div class="row">
+    <div class="col-md-4">
+        @using (Html.BeginForm("Post", "Payment", FormMethod.Post, new { id = "payment-form" }))
```

Comment:
Would be nicer to use a standard `<form>` tag and the new tag helpers. See https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/intro?view=aspnetcore-2.1